src/Route.php

```diff
@@ -139,6 +146,19 @@ public static function options(string $pattern, ?MiddlewareDispatcher $dispatche
         return self::methods([Method::OPTIONS], $pattern, $dispatcher);
     }
 
+    /**
+     * @param string $pattern
+     *
+     * @return self
+     */
+    public static function static(string $pattern): self
```

Comment:
There is no need for a separate method. Generation-only route is the one that doesn't have action.